projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -43,25 +46,58 @@ public static void saveAgency(Path csvPath, String path) {
           isHeader = false;
           continue;
         }
-        Agency.Builder agency = Agency.newBuilder();
-        agency
-            .setAgencyId(Integer.parseInt(record.get("agency_id")))
-            .setAgencyName(record.get("agency_name"))
-            .setAgencyUrl(record.get("agency_url"))
-            .setAgencyTimezone(record.get("agency_timezone"))
-            .setAgencyLang(record.get("agency_lang"))
-            .setAgencyPhone(record.get("agency_phone"))
-            .setAgencyFareUrl(record.get("agency_fare_url"))
-            .build()
-            .writeDelimitedTo(output);
+        if (record.get("agency_id").equals(ISRAEL_RAIL_AGENCY_ID)) {
```

Comment:
I don't think we need the Agency file actually. It will only have 1 line and it will be constant.